app/frontend/src/components/Team.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-container>
+    <p class="text-right">
+      <strong>
+        <v-icon class="pr-2">group</v-icon>Team members
+        <span v-if="usersLoaded">({{users.length}})</span>
```

Comment:
Style: `{{ users.length }}` Should have spaces between the brackets and contents to make it easier to catch visually.

Reply:
Fixed